.github/workflows/main.yml

```diff
@@ -10,21 +10,21 @@ jobs:
       fail-fast: false
       matrix:
         node-version:
-          - 16
-          - 14
-          - 12
+          - 18.0.0
+          - 16.4.0
+          - 14.18.0
```

Comment:
I would prefer using non-specific minors so it upgrades when new minor versions are out.

Reply:
Fixed!